client/src/pages/VotePage.tsx

```diff
@@ -15,6 +15,7 @@ export const VotePage = () => {
   const cookies = new Cookies();
   const [pollCode] = useState(cookies.get(pollCodeCookie));
   const [started, setStarted] = useState(false);
+  const [notifPermission, setNotifPermission] = useState("denied");
```

Comment:
I would probably change this to use a boolean and rename the state variables/function for it

Reply:
+1

Reply:
Changed as requested (to `hasAllowedNotif`).